### PR TITLE
PickerRow for multiple components

### LINKED ISSUE #1237: How to create a PickerRow with multiple components
It is posible create a PickerRow or PickerInlineRow with multiple components?

### LINKED ISSUE #1533: How can I handle nested options with Eureka (with PushRow or PickerRow)
I want to find a general solution to handle nested options nicely (e.g. select the country first and select the city, from json data like    `{
      "Afghanistan": [
        "Herat",
        "Kabul",
        "Kandahar",
        ...
      ],
      "Albania": [
        "Elbasan",
        "Petran",
        "Pogradec",
        ...
      ],
      ...
   }`).  However `options` can only be assign with items array.  Is there any clean and nice solution to this problem?

### PR DESCRIPTION
**Pull request by mats-claassen**
# WIP:

- [x] PickerRow for 2 components
- [x] PickerInlineRow for 2 components
- [x] PickerInputRow for 2 components
- [x] PickerRow for 3 components
- [x] PickerInlineRow for 3 components
- [x] PickerInputRow for 3 components

Fixes #1533 
Fixes #1237

### CODE REVIEW

**Comment by mats-claassen** [comment]
Any comments on this @mtnbarreto ?

**Comment by mats-claassen** [comment]
Finished the pickers. 

There is a lot of repeated code for example `TriplePickerRow` and `TriplePickerInputRow` are very similar but have a different base class and override methods. Any suggestion is welcome

File: Source/Rows/PickerRow.swift [hunk 1]
```diff
@@ -67,6 +67,10 @@ open class PickerCell<T> : Cell<T>, CellType, UIPickerViewDataSource, UIPickerVi
         textLabel?.text = nil
         detailTextLabel?.text = nil
         picker.reloadAllComponents()
+        selectValue()
```
Comment (mtnbarreto) [inline Source/Rows/PickerRow.swift]: I would prefer to create a _PickerCell class and  override `update()` method for each case.

**Comment by dbmrq** [comment]
I just tried to use this and…

captura de tela 2018-05-17 as 00 22 13

captura de tela 2018-05-17 as 00 26 34

Am I doing something wrong?

**Comment by dbmrq** [comment]
Oh, never mind!

In case anybody else is wondering, you have to specify the parameters like `DoublePickerInlineRow<Double, Double>("timeRow")`. It would be nice if this were documented somewhere. Or is it, and I just missed it?

**Pull request merged**